includes/class-sensei-course.php

```diff
@@ -3646,6 +3646,9 @@ public function remove_legacy_course_actions() {
 		// Take this course.
 		remove_action( 'sensei_single_course_content_inside_before', [ __CLASS__, 'the_course_enrolment_actions' ], 30 );
 
+		// Add message links to courses.
+		remove_action( 'sensei_single_course_content_inside_before', [ Sensei()->post_types->messages, 'send_message_link' ], 35 );
+
 		// @todo Remove additional actions from `\Sensei_Course::add_legacy_course_hooks` as implemented in blocks.
```

Comment:
I think now all of the blocks are hidden. So I think we can remove this comment. =)

And maybe we could also add the contact block to the `Sensei_Course::is_legacy_course` array to consider it a new course when this block is added too.

Reply:
👍  1b2fab2 f4fe35b